tests/test_heat.py

```diff
@@ -92,6 +115,20 @@ def heat_network_options(self):
             return options
 
     def test_min_max_pressure_options(self):
+        """
+        This test is to check if the min and max pressure options are correctly enforced on the
+        optimization. This is achieved by creating an problem where the pressure drop needed
```

Comment:
" ... a problem ..." instead of " ... an problem ..."

Reply:
done